### PR TITLE
PAS-452 | Matching documentation for e-mail provider fallback

### PR DESCRIPTION
**Pull request by jonashendrickx**
### Ticket

- Closes [PAS-452](https://bitwarden.atlassian.net/browse/PAS-452)

### Checklist

I did the following to ensure that my changes were tested thoroughly:

- \_\_

I did the following to ensure that my changes do not introduce security vulnerabilities:

- \_\_

[PAS-452]: https://bitwarden.atlassian.net/browse/PAS-452?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/guide/self-hosting/configuration.md [hunk 1]
```diff
@@ -63,85 +63,201 @@ By default, the container will use Sqlite if nothing else is specified. The data
...
 ```bash
-docker exec -it {name-of-container} cat /app/AdminConsole/mail.md
+docker exec -it {name-of-container} cat /app/mail.md
+```
+
+### JSON
+
+Reference: [Configuration in ASP.NET Core](https://learn.microsoft.com/en-us/aspnet/core/fundamentals/configuration/?view=aspnetcore-8.0)
+
+What is important is to configure ‘Mail**From' as shown below. This is required to have a fallback e-mail address to send e-mails from. On the 'Mail**Providers’ is an array, which is an ordered list of e-mail providers that we will attempt to execute in order if they fail. To configure an e-mail provider, see the sub sections below.
```
Comment (Tyrrrz) [inline src/guide/self-hosting/configuration.md:89]: 1. Inconsistent backticks/quotes. Converter to backticks as it seems like it makes sense to use an inline code block here.
2. I think `.` and `:` are more commonly used as delimiters for nested JSON.

```suggestion
What is important is to configure `Mail.From` as shown below. This is required to have a fallback e-mail address to send e-mails from. On the `Mail.Providers` is an array, which is an ordered list of e-mail providers that we will attempt to execute in order if they fail. To configure an e-mail provider, see the sub sections below.
```

File: src/guide/self-hosting/configuration.md [hunk 2]
```diff
@@ -63,85 +63,201 @@ By default, the container will use Sqlite if nothing else is specified. The data
...
+  "SslOverride": false,
+  "TrustServer": true
+}
+```
+
+### Environment variables
+
+Reference: [Configuration in ASP.NET Core](https://learn.microsoft.com/en-us/aspnet/core/fundamentals/configuration/?view=aspnetcore-8.0)
+
+What is important is to configure ‘Mail**From' as shown below. This is required to have a fallback e-mail address to send e-mails from. On the 'Mail**Providers’ is an array, which is an ordered list of e-mail providers that we will attempt to execute in order if they fail. To configure an e-mail provider, see the sub sections below.
```
Comment (Tyrrrz) [inline src/guide/self-hosting/configuration.md:173]: This seems like a duplicate paragraph from the one above, not sure if intended

```suggestion
What is important is to configure `Mail.From` as shown below. This is required to have a fallback e-mail address to send e-mails from. On the `Mail.Providers` is an array, which is an ordered list of e-mail providers that we will attempt to execute in order if they fail. To configure an e-mail provider, see the sub sections below.
```

File: src/guide/self-hosting/configuration.md [hunk 3]
```diff
@@ -63,85 +63,201 @@ By default, the container will use Sqlite if nothing else is specified. The data
...
+```
 
 ## config.json
 
 :::warning
 Requirements:
+
 - Persistent storage, see 'Volumes'.
-:::
+  :::
```
Comment (Tyrrrz) [inline src/guide/self-hosting/configuration.md:246]:
```suggestion
:::
```

File: src/guide/self-hosting/configuration.md [hunk 4]
```diff
@@ -63,85 +63,201 @@ By default, the container will use Sqlite if nothing else is specified. The data
...
 - Passwordless::ApiKey
 - Passwordless::ApiSecret
 - PasswordlessManagement::ManagementKey
 - SALT_KEY
 
 It is recommended that you have them generated automatically, the first time you run `bitwarden/passwordless-self-host`.
+
+```
+
 ```
```
Comment (Tyrrrz) [inline src/guide/self-hosting/configuration.md:263]: Looks like an empty code block

```suggestion
```

File: src/guide/self-hosting/configuration.md (hunk 3 above)
Comment (jonashendrickx) [reply src/guide/self-hosting/configuration.md:247]: This was done by prettier

**Review by abergs — APPROVED** [review]

**Pull request merged**